app/src/main/java/com/duckduckgo/app/tabs/model/TabRepository.kt

```diff
@@ -29,11 +29,11 @@ interface TabRepository {
     /**
      * @return tabId of new record
      */
-    suspend fun add(url: String? = null, skipHome: Boolean = false, isDefaultTab: Boolean = false): String
```

Comment:
I tried to simplify the overhead of thinking about `isDefaultTab` and in the code using these methods. `isDefaultTab` was `true` only in one place out of many different ones.

Reply:
👍